pkg/services/ngalert/api/tooling/definitions/alertmanager.go

```diff
@@ -587,10 +588,18 @@ func (c *GettableApiAlertingConfig) validate() error {
 
 // Config is the top-level configuration for Alertmanager's config files.
 type Config struct {
-	Global       *config.GlobalConfig  `yaml:"global,omitempty" json:"global,omitempty"`
-	Route        *Route                `yaml:"route,omitempty" json:"route,omitempty"`
-	InhibitRules []*config.InhibitRule `yaml:"inhibit_rules,omitempty" json:"inhibit_rules,omitempty"`
-	Templates    []string              `yaml:"templates" json:"templates"`
+	Global            *config.GlobalConfig       `yaml:"global,omitempty" json:"global,omitempty"`
+	Route             *Route                     `yaml:"route,omitempty" json:"route,omitempty"`
+	InhibitRules      []*config.InhibitRule      `yaml:"inhibit_rules,omitempty" json:"inhibit_rules,omitempty"`
+	MuteTimeIntervals []InternalMuteTimeInterval `yaml:"mute_time_intervals,omitempty" json:"mute_time_intervals,omitempty"`
+	Templates         []string                   `yaml:"templates" json:"templates"`
+}
+
+// InternalMuteTimeInterval is a workaround till https://github.com/prometheus/alertmanager/pull/2765
```

Comment:
Looks like https://github.com/prometheus/alertmanager/pull/2765 was merged, can we use `MuteTimeInterval`?

Reply:
I think they'll need to tag a new version first.

Reply:
NVM will update it using the commit-hash as we already use a commit-hash.